test/gui/tst_syncing/test.feature

```diff
@@ -7,7 +7,7 @@ Feature: Syncing files
     Background:
         Given user "Alice" has been created on the server with default attributes and without skeleton files
 
-    @smokeTest @skip @issue-9281
```

Comment:
Should we remove the issue tag or keep it for future references?

Reply:
I am not sure. What do you suggest?

Reply:
Let's not remove the issue tag as we do in other repos

Reply:
ok, done